homeassistant/components/device_tracker/ble_tracker.py

```diff
@@ -0,0 +1,90 @@
+"""Tracking for bluetooth devices."""
+import logging
+from datetime import timedelta
+
+from homeassistant.helpers.event import track_point_in_utc_time
+from homeassistant.components.device_tracker import (
+    YAML_DEVICES,
+    CONF_TRACK_NEW,
+    CONF_SCAN_INTERVAL,
+    DEFAULT_SCAN_INTERVAL,
+    load_config,
+)
+import homeassistant.util as util
+import homeassistant.util.dt as dt_util
+
+_LOGGER = logging.getLogger(__name__)
+
+REQUIREMENTS = ['gattlib']
+
+BLE_PREFIX = 'BLE_'
+
+def setup_scanner(hass, config, see):
+    """Setup the BLE Scanner."""
```

Comment:
I do not like this name. It is an abbreviation but it means the same as the other tracker, that will be confusing.

We can rename it bluetooth_tracker_bluez, that way it is clear what is being used.